### ISSUE TITLE
Bitcoin Core v0.16: createmultisig API changes

### ISSUE THREAD

**Issue by bitcoinhodler** [GlacierProtocol/GlacierProtocol#17]
# Problem

The API used by `glacierscript.py` for `createmultisig` has been deprecated in Bitcoin Core v0.16.0rc1 and will be removed in v0.17. Running create-deposit-data in v0.16.0rc1 results in this error:

```
    error code: -5
    error message:
    Invalid public key: mn1f471kQHYx9oUnwc4TUy9KzbMx9847Dk
    Note that from v0.16, createmultisig no longer accepts addresses. Clients must transition to using addmultisigaddress to create multisig addresses with addresses known to the wallet before upgrading to v0.17. To use the deprecated functionality, start bitcoind with -deprecatedrpc=createmultisig
    Traceback (most recent call last):
      File "./glacierscript.py", line 796, in <module>
          deposit_interactive(args.m, args.n, args.dice, args.rng)
            File "./glacierscript.py", line 595, in deposit_interactive
            "bitcoin-cli -testnet -datadir=bitcoin-test-data createmultisig {0}".format(argstring), shell=True)
              File "/usr/lib/python2.7/subprocess.py", line 573, in check_output
                  raise CalledProcessError(retcode, cmd, output=output)
                  subprocess.CalledProcessError: Command 'bitcoin-cli -testnet -datadir=bitcoin-test-data createmultisig 2 '["mn1f471kQHYx9oUnwc4TUy9KzbMx9847Dk", "mxtjsVSypDUjWAvc5nNRMEqYY5C4G2HpWL", "mwLScBiargpRuEJJo9zkrCrcHkb23qJp4m", "muYPF5rfGfqM1ucX5BCcYpRLx1y2BJmYbC"]'' returned non-zero exit status 5
```

# Impact
Without a fix, any APP USBs made using this (future) Bitcoin Core will not be able to create deposit data. Withdrawals still work, after fixing the previous issue (#16).

# Solution
The error message shows a simple, short-term workaround: add `-deprecatedrpc=createmultisig`.

The full fix will involve a more significant change to use the new API, which will be required for Bitcoin Core v0.17.

**Comment by bitcoinhodler** [comment]
Thoughts on transition to new API:

It would be a pain to use `addmultisigaddress` since the results differ between v0.15.1 and v0.16.0rc1.

We could continue to use `createmultisigaddress` but use hex-encoded public keys instead of bitcoin addresses.

Or, just wait until v0.16.0 release is available from the PPA and assume any new APP USBs created will use that version or newer. Once that threshold is reached, I will be changing all this code anyway, to create segwit addresses.

**Comment by bitcoinhodler** [comment]
Fixed in v0.92
